CONTRIBUTING.rdoc

```diff
@@ -23,13 +23,23 @@ here: http://guides.rubygems.org/contributing/
 
 == Getting Started
 
-Run:
+==== To Run Tests:
```

Comment:
I think this should be `=== Setup the Development Environment`

As a side-effect it happens to run tests, but you don't need to run `rake newb` to run the tests when you make changes.